src/itest/java/uk/gov/companieshouse/insolvency/data/controller/HealthCheckControllerITest.java

```diff
@@ -1,26 +0,0 @@
-package uk.gov.companieshouse.insolvency.data.controller;
```

Comment:
Can you please confirm why you deleted this file?

Reply:
It was verifying same as new Cucumber feature file iTest, I think we added this test as PoC in the beginning